.travis.yml

```diff
@@ -0,0 +1,4 @@
+jobs:
+  include:
+    - stage: build and push container image
+      script: bin/build_container_image
```

Comment:
I'm good with this for now, but ideally this should be split into 2 separate parts.

The primary job should just be to build and tag.  This can (must?) build on PRs.
The second half should be a deploy step, and that should only run on master.

Reply:
- The build and push need to happen in the same "stage" since different stages get their own VM, so we wouldn't have access to anything built in the first stage.  That's why I put that logic in the script rather than in the `.travis.yml`.
- I thought it would still be a good idea to only build on PRs so that we catch any Dockerfile errors before it's merged.

Reply:
Ah that's frustrating

:+1:

Reply:
Switched to docker.io with the bot user and a token